pkg/resource/cluster/hooks.go

```diff
@@ -30,8 +30,9 @@ import (
 )
 
 var (
-	condMsgCurrentlyDeleting     = "cluster currently being deleted"
-	condMsgNoDeleteWhileUpdating = "cluster is being updated. cannot delete"
+	condMsgCurrentlyDeleting            = "cluster currently being deleted"
+	condMsgNoDeleteWhileUpdating        = "cluster is being updated. cannot delete"
```

Comment:
Can we make messages a bit consistent? `cluster currently being updated`

Reply:
These two variables are not related to flaky tests. I didn't modify or create them in this PR, so I will mark it and change it in a new PR.

Reply:
Yes please. The english grammar for these two messages should be improved. `cluster is currently being deleted` and `cluster is currently being updated, cannot delete` are my suggestions